src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -163,7 +164,7 @@
       StartWorkflowExecutionRequest startRequest,
       String runId,
       Optional<TestServiceRetryState> retryState,
-      int backoffStartIntervalInSeconds,
+      long backoffStartIntervalInSeconds,
```

Comment:
change from long to Duration?

Reply:
Either change to the duration or make sure that all the intervals are in milliseconds.